DecipherNow/gm-fabric-dashboard
# fix: add margin to prevent lapping

Resolves #1374 

## How to Test

  - Go to Explorer
  - Tab to search bar
  - Check hover and active states for lapping
